### ISSUE TITLE
Typo in Ezekiel 40:31 and 40:37 - found the words 'utter court ' used instead of 'outer court'

### ISSUE THREAD

**Issue by LAfricain** [BibleCorps/ENG-B-AKJV2018-pd-PSFM#8]
I have found in AKJV  Ezekiel 40:31 and 40:37 - found the words 'utter court ' used instead of 'outer court'.

**Comment by cmahte** [comment]
**_All_** occurrences of **utter court** in Ezekiel should read **_outer court_**.  This occurs in:

_Ezekiel 40:31
Ezekiel 40:37
Ezekiel 42:1
Ezekiel 42:3
Ezekiel 42:7
Ezekiel 42:8
Ezekiel 42:9
Ezekiel 42:14
Ezekiel 44:19
Ezekiel 46:20
Ezekiel 46:21_


_Ezekiel 47:2_ **utter gate** should read **_outer gate_**.

**Comment by cmahte** [comment]
And from a query i made, editions of the KJV have used both "outer" and "outward". 

[Rick Norris](https://www.facebook.com/groups/4001886615/user/100088773404566/?__cft__[0]=AZVHbJMS5q7YnM62Na1dWFHd3fmvcH8_MnRvHRk9iwCph6o5r5CzsoLm2RK-rx5eRHqiHqPUKVY020l4OoORd1o-vGR9YQBCEZA5TxU12zcOiCR5zQBRr7Jgwpz5rivMm_h_LxKLcNm6nl56qQVY6_StjO_f-pHjfMfkbWr8JF8lcQ&__tn__=R]-R)
There are some KJV editions that updated "utter" to "outer" at those verses as early as the 1700's just as it had been updated at Ezekiel 10:5. These may be considered two different words in English today, but back in 1611 it was probably or possibly a spelling variation. Most present KJV editions still have "utter" at Ezekiel 40:31 besides the 1975 Open Bible, the American Bible Society KJV editions, and the 2005 and 2011 Cambridge by David Norton.
Ezekiel 10:5 [see Ezek. 40:31, 42:1, 46:21] [utter court--1560 Geneva]
utter court (1675, 1679, 1681, 1709, 1713, 1715, 1720, 1737, 1743, 1747, 1754, 1758, 1762, 1765 Oxford) [1629, 1635, 1637, 1638, 1648, 1683, 1816, 1817, 1873 Cambridge] {1611, 1613, 1614, 1616, 1617, 1626, 1630, 1631, 1634, 1640, 1644, 1648, 1652, 1655, 1657, 1660, 1672, 1684, 1693, 1698, 1703, 1706, 1711, 1717, 1730, 1735, 1741, 1743, 1767 London} (1755 Oxon) (1637, 1638, 1715, 1722, 1735, 1751 Edinburgh) (1762 Dublin) (1645 Dutch) (1700 MP) (1746 Leipzig) (1776 Pasham) (1821 Brown) (1824, 1826 Bagster) (1840 Roby) (2000, 2002 ZOND) (HPB) (2007, 2008, 2010, 2011 HEND) (NHPB)
outward court (1813 Carey)
outer court (1768, 1769 Oxford, SRB) [1747, 1756, 1760, 1762, 1765, 1769, 2005, 2011 Cambridge, DKJB] {1747, 1760, 1763, 1764, 1772 London} (1756, 1764, 1766, 1769, 1789, 1793, 1810 Edinburgh) (1782 Aitken)
Ezekiel 40:31 [see Ezek. 10:5] [see outward court at Ezek. 40:17, 20, 34]
outer court (1768, 1772, 1777 Oxford) [2005, 2011 Cambridge] {1747, 1750, 1759, 1760, 1763, 1764, 1768, 1772, 1853, 1879 London} (1756, 1760, 1764, 1766, 1769, 1787, 1789, 1791, 1793, 1810, 1820, 1827, 1842, 1843, 1858 Edinburgh) (1860, 1866 Glasgow) (1801, 1809 Dublin) (1774 Fortescu) (1777 Wood) (1782 Aitken) (1790 Bolton) (1790, 1804, 1828 MH) (1791, 1816 Collins) (1799 Helston) (1801 Hopkins) (1802, 1813 Carey) (1803 Etheridge) (1804 Blomfield) (1804 Gower) (1807, 1813 Johnson) (1809, 1810, 1813, 1818, 1826, 1828 Boston) (1810, 1814 Woodward) (1814, 1832, 1835, 1858 Scott) (1815 Walpole) (1816 Albany) (1816, 1836, 1848 Hartford) (1818 Holbrook) (1818, 1819, 1827, 1829, 1843, 1851, 1853, 1854, 1855, 1858, 1868, 1888, 1894, 1902, 1954, 1956, 1957, 1963, 1968, 1970, 1971, 1984, 1988, 2004, 2008 ABS) (1827 Smith) (1831 Brown) (1832 PSE) (1834 Coit) (1835 Towar) (1836 Stebbing) (1840 Roby) (1841 Thomas) (1843, 1850 AFBS) (1843 Robinson) (1846 Benson) (1846 Portland) (1845, 1854, 1857, 1876 Harding) (1876 Porter) (1910 Collins) (1911 TCE) (1924, 1958 Hertel) (1945 World) (1948 WSE) (1968 Royal) (1970 TN) (1975 Open) (RRB) (WMCRB) (1984 AMG) (1987 IBS) (LASB) (2006 PENG) (1833 WEB) (1842 Bernard)
utter court (1769 Oxford, SRB) [1629, 1769 Cambridge, DKJB]
Ezekiel 42:1 [see Ezek. 10:5 and 1611 chapter heading “outward court”]
outer court (1675, 1768 Oxford) [2005, 2011 Cambridge] {1630, 1747, 1750, 1759, 1760, 1763, 1764, 1768, 1772, 1853, 1879 London} (1756, 1760, 1764, 1766, 1769, 1787, 1789, 1791, 1793, 1810, 1820, 1827, 1842, 1843, 1858 Edinburgh) (1860, 1866 Glasgow) (1801, 1809 Dublin) (1782 Aitken) (1790 Bolton) (1790, 1804, 1828 MH) (1791, 1816 Collins) (1799 Helston) (1801 Hopkins) (1803 Etheridge) (1804 Blomfield) (1807, 1813 Johnson) (1809, 1818, 1826, 1828 Boston) (1810, 1814 Woodward) (1813 Carey) (1814, 1832, 1835, 1858 Scott) (1815 Walpole) (1816 Albany) (1816, 1836, 1848 Hartford) (1818 Holbrook) (1818, 1819, 1827, 1829, 1843, 1851, 1853, 1854, 1855, 1858, 1868, 1888, 1894, 1902, 1954, 1956, 1957, 1963, 1968, 1970, 1971, 1984, 1988, 2004, 2008 ABS) (1827 Smith) (1831 Brown) (1832 PSE) (1834 Coit) (1835 Towar) (1836 Stebbing) (1841 Thomas) (1843, 1850 AFBS) (1843 Robinson) (1846 Benson) (1846 Portland) (1845, 1854, 1857, 1876 Harding) (1876 Porter) (1910 Collins) (1911 TCE) (1924, 1958 Hertel) (1945 World) (1948 WSE) (1968 Royal) (1970 TN) (1975 Open) (1976 Holman) (CSB) (RRB) (WMCRB) (LASB) (1984 AMG) (2006 PENG) (2002, 2010 KJVER) (1833 WEB) (1842 Bernard)
utter court (1769 Oxford, SRB) [1629, 1769 Cambridge, DKJB]

**Comment by cmahte** [comment]
I will attempt to reach out to Mrs. Engelbrite about this. 

The problem is that the list of changes doesn't include word separations like this. 'utter' in the AKJV has 3 meanings: 'speak' or some similar meaning, 'complete', 'end', or 'highest' or similar... and here 'exterior side' which is related to the 2nd, but has become unreadable since 1611.  So I'm struggling more with how to document this properly than whether or not to change it.   Maybe there are other cases in the word list where the word isn't universally changed, and it the list is just an indication.

**Comment by cmahte** [comment]
Mrs. Engelbrite's reply: 

Hello, Michael,

Thank you for contacting me with these errors. I have made corrections in the printed version, and they should take effect by the new year on Amazon. In the word list I wrote "**utter (in Ezekiel 40-47), outer**".

I have also copied the errors and sent them to my husband to see if he will correct them in the online version. He suffered a stroke in 2019, but still manages to program a bit.

Thank you for all your efforts. God bless you.
Eve Engelbrite
